.github/CODEOWNERS

```diff
@@ -206,7 +206,7 @@
 
 # Scene
 
-/scene/                                       @godotengine/_systems @godotengine/core
+/scene/                                       @godotengine/_systems
```

Comment:
I'm not too fond of the `_systems` catchall.

The groups with `_` were not meant to be used as teams, as they are collections of people with completely unrelated areas of expertise. Asking them all to deal with what's unspecified generates a lot of noise.

I think we should remove those catchalls and assign PRs lacking reviewers manually (and eventually improve the `CODEOWNERS` to handle those missing pieces). Or as suggested on RC, we could make a dummy fallback team for those, and then go through this team's review requests once in a while to assign them better.

Reply:
Updated to get rid of the leading-underscore teams entirely. Now all fallbacks are assigned to the new, dummy team @godotengine/fallback.